components/aiplatform-components/tests/test_lookup_model.py

```diff
@@ -18,7 +18,14 @@
 import pytest
 
 
-def test_lookup_model(tmpdir):
+@pytest.fixture
```

Comment:
do we need this as a fixture? Why not just import the `python_func` instead of the component for the purposes of the unit test?